### PR TITLE
Add introductory hello-world exercise

### PR DESCRIPTION
**Pull request by ramonh**
This addresses exercism/rossconf#2 towards goal [#12](exercism/rossconf#12).

The hello-world exercise has three test cases, one for no name given, one for the name 'Alice' given and one for the name 'Bob' given. This is to avoid the hardcoding of a name into the solution.

### CODE REVIEW

**Comment by kytrinyx** [comment]
This looks great. I'm going to hold off on merging it until we have the code in place to ignore hello-world in the nitpicking list, and to submit an encouraging comment by rikki-.

Would you submit a separate PR with the `SETUP.md`? That would be useful to have even before hello world goes live.

**Comment by ramonh** [comment]
Thanks for the feedback! I've removed the SETUP.md commit.

**Comment by kytrinyx** [comment]
:heart: here we go!

**Comment by ramonh** [comment]
Wahey! Awesome, still getting you that Obj-C version. :smile:

**Comment by kytrinyx** [comment]
Niiiice!

File: hello-world/example.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class HelloWorld
+  def self.hello(name = '')
+    name == '' ? 'Hello, world!' : "Hello, #{name}!"
```
Comment (kotp) [inline hello-world/example.rb]: This is a little bit "tricky". You already have a default value, why not just use it?

``` ruby
def self.hello(name = 'World')
  "Hello, #{name}!""
end
```

Least amount of complication.

**Pull request merged**